src/interaction/InteractionManager.js

```diff
@@ -263,20 +263,32 @@ export default class InteractionManager extends EventEmitter
         this.processTouchMove = this.processTouchMove.bind(this);
 
         /**
-         * Every update cursor will be reset to this value, if some element wont override it in
-         * its hitTest.
+         * Dictionary of how different cursor modes are handled. Strings are handled as CSS cursor
+         * values, objects are handled as dictionaries of CSS values for interactionDOMElement,
+         * and functions are called instead of changing the CSS.
+         * Default CSS cursor values are provided for 'default' and 'pointer' modes.
+         * TODO: Finish documenting this.
+         */
+        this.cursorStyles = {
+            default: 'inherit',
+            pointer: 'pointer',
+        };
+
+        /**
+         * The mode of the cursor that is being used.
+         * The value of this is a key from the cursorStyles dictionary.
```

Comment:
I'm not sure how to properly document this. It is a dictionary/map of the form `<string, string|Function|{[cssProp:string]:string}>`.